Ladybird/AppKit/Application/ApplicationDelegate.mm

```diff
@@ -355,9 +357,79 @@ - (NSMenuItem*)createDebugMenu
     auto* menu = [[NSMenuItem alloc] init];
     auto* submenu = [[NSMenu alloc] initWithTitle:@"Debug"];
 
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump DOM Tree"
+                                                action:@selector(dumpDOMTree:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Layout Tree"
+                                                action:@selector(dumpLayoutTree:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Paint Tree"
+                                                action:@selector(dumpPaintTree:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Stacking Context Tree"
+                                                action:@selector(dumpStackingContextTree:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Style Sheets"
+                                                action:@selector(dumpStyleSheets:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump All Resolved Styles"
+                                                action:@selector(dumpAllResolvedStyles:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump History"
+                                                action:@selector(dumpHistory:)
+                                         keyEquivalent:@""]];
     [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Cookies"
                                                 action:@selector(dumpCookies:)
                                          keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Local Storage"
+                                                action:@selector(dumpLocalStorage:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[NSMenuItem separatorItem]];
+
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Show Line Box Borders"
+                                                action:@selector(toggleLineBoxBorders:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[NSMenuItem separatorItem]];
+
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Collect Garbage"
+                                                action:@selector(collectGarbage:)
+                                         keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump GC graph"
```

Comment:
I know this is copy-pasted from the Qt chrome, but let's capitalize "Graph". It stuck out like a sore thumb when I was playing around with this patch :)